synapse/__init__.py

```diff
@@ -21,6 +21,7 @@
 # the built-in cortex modules...
 BASE_MODULES = (
     ('synapse.models.syn.SynMod', {}),
+    ('synapse.models.axon.AxonMod', {}),
```

Comment:
I'm not certain if we want to include the axon stuff in the default base modules; or if its something we want to include in the axon-specific sqlite core which is created and loaded here https://github.com/vertexproject/synapse/blob/master/synapse/axon.py#L453